app/views/providers/applicants/show.html.erb

```diff
@@ -13,18 +13,19 @@
       ) do |form| %>
 
     <div class="govuk-!-padding-bottom-2"></div>
-
-    <%= form.govuk_text_field :first_name, class: 'govuk-!-width-one-half' %>
-    <%= form.govuk_text_field :last_name, class: 'govuk-!-width-one-half' %>
+    <%= govuk_bold_label t('.full_name') %>
```

Comment:
OK - this is a difficult one and you may have to do something strange here. Basically, you have one label for two fields. Looking at [this Stackoverflow answer](https://stackoverflow.com/a/31274644), I think the best option would be to use a span and add `arel-label` entries for the inputs:
```
<%= content_tag :span, t('.full_name'), class: 'govuk-fieldset__legend govuk-fieldset__legend--m') %>
<%= form.govuk_text_field :first_name, class: 'govuk-input govuk-!-width-three-quarters', arel_label: :first_name %> 
<%= form.govuk_text_field :last_name, class: 'govuk-input govuk-!-width-three-quarters', arel_label: :last_name: :name %>
```
I think the `arel_label` options will be convert to `arel-label` in the output but check and if not you'll need to use the `'arel-label' => :last_name` form.

Reply:
Also this may be a simpler style to use for the first line if you prefer:
```html
<span class="govuk-fieldset__legend govuk-fieldset__legend--m"><%= t('.full_name') %></span>
```